Sources/RxCodable/ObservableType+RxCodable.swift

```diff
@@ -17,3 +17,47 @@ public extension ObservableType where E == String {
       .map(type, using: decoder)
   }
 }
+
+public extension ObservableType where E == [String: Any] {
+  public func map<T>(_ type: T.Type, using decoder: JSONDecoder? = nil) -> Observable<T> where T: Decodable {
+    return self
+      .map { dict in try JSONSerialization.data(withJSONObject: dict) }
+      .map(type, using: decoder)
+  }
+}
+
+public extension ObservableType where E == [[String: Any]] {
+  public func map<T>(_ type: Array<T>.Type, using decoder: JSONDecoder? = nil) -> Observable<[T]> where T: Decodable {
```

Comment:
What is this? The 'array mapping' I meant was about `func mapArray()` which maps `Observable<[T]>` to `Observable<[[String: Any]]>`.

Reply:
Oh.. my misunderstood. now I see.

Reply:
Hmm.. It's not as easy as I thought.